### PR TITLE
chore: cleanup CLI help and config

### PR DESCRIPTION
**Pull request by rawkode**

### CODE REVIEW

File: cmd/blox_build.go [hunk 1]
```diff
@@ -25,14 +25,21 @@ var (
...
 	Use:   "build",
-	Short: "Validate and build your data",
+	Short: "Validate & Build",
+	Long: `The build command will ensure that your content is correct by
+validating it against your schemata. Once validated, it will render all
+your content into a single JSON file, which can be consumed by your tooling
+of choice.
+
+Referential Integrity can be enforced with -i. This ensures that any fields
+ending with _id are valid references to identifies within the other content type.`,
```
Comment (bketelsen) [inline cmd/blox_build.go:42]: typo: identities?

File: blox.cue [hunk 2]
```diff
@@ -1,5 +1,4 @@
 {
-    data_dir: "dogfood/data/source"
-    schema_dir: "dogfood/data/schema"
-    build_dir: "_build"
+    data_dir: "dogfood/data"
```
Comment (bketelsen) [inline blox.cue:2]: our existing dogfood content is in `dogfood/data/source` are you intending to move this as part of the PR?

File: cmd/blox_new.go [hunk 3]
```diff
@@ -23,13 +23,10 @@ var (
...
-	Short: "Create a new content file",
-	Long: `A longer description that spans multiple lines and likely contains examples
-and usage of using your command. For example:
-
-Cobra is a CLI library for Go that empowers applications.
-This application is a tool to generate the needed files
-to quickly create a Cobra application.`,
+	Short: "Create a new content file for the target dataset",
+	Long: `This command will allow you to create new content based on the
+template attributes within the schemata. By providing a dataset name and ID
```
Comment (bketelsen) [inline cmd/blox_new.go:28]: maybe `dataset name and ID(slug)` to make the ID term more friendly to our usage.

File: cmd/remote_list.go [hunk 4]
```diff
@@ -12,20 +12,15 @@ import (
 
 // listCmd represents the list command
 var remoteListCmd = &cobra.Command{
-	Use:   "list",
+	Use:   "list <remote repository URL>",
 	Short: "List schemas and versions available in a remote repository",
```
Comment (bketelsen) [inline cmd/remote_list.go:16]: list schemata

**Review by bketelsen — APPROVED** [review]
small changes requested, otherwise LGTM

File: blox.cue (hunk 2 above)
Comment (rawkode) [reply blox.cue:2]: I did move it, so it aligned with the defaults ... I'm sure I did 😓

**Pull request merged**